.circleci/config.yml

```diff
@@ -11,15 +11,12 @@ jobs:
           command: |
             brew install carthage || true
             brew outdated carthage || brew upgrade carthage
-      - run:
-          name: Install SwiftLint
-          command: wget --output-document /tmp/SwiftLint.pkg https://github.com/realm/SwiftLint/releases/download/0.26.0/SwiftLint.pkg && sudo installer -pkg /tmp/SwiftLint.pkg -target /
```

Comment:
We will still need swiftlint, just need to include it in another way since circleci works with jobs for that

Reply:
Yeah, just removed it to see if it works otherwise. Should be added in a lint job.